.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+  - package-ecosystem: 'github-actions'
+    directory: '/'
```

Comment:
should this be limited to the .github directory or doesn't it matter much

Reply:
It looks for a `.github/workflows` subdirectory in the path given